ansible/roles/ooni-backend/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+enable_dns_helper: yes
+enable_dns_discovery_helper: yes
+enable_http_return_json_headers_helper: yes
+enable_tcp_echo_helper: yes
+enable_daphn3_helper: yes
+
+dns_resolver_address: "8.8.8.8:53"
+dns_test_helper_address: "37.218.247.110:57004"
+http_return_json_headers_address: "http://37.218.247.95"
+ssl_address:
+tcp_echo_address: "37.218.247.110"
+traceroute_address: "199.119.112.49"
+http_return_json_headers_port: 57001
+tcp_echo_port: 57002
+daphn3_port: 57003
+dns_port_tcp: 57004
+dns_port_udp: 57004
```

Comment:
This should rather use the default values: https://github.com/TheTorProject/ooni-backend/blob/master/oonib.conf.example#L60.
`57004/TCP 57005/UDP`